adminSiteServer/apiRoutes/chartViews.ts

```diff
@@ -109,30 +115,28 @@ getRouteWithROTransaction(apiRouter, "/chartViews", async (req, res, trx) => {
     }))
 
     return { chartViews }
-})
-
-getRouteWithROTransaction(
-    apiRouter,
-    "/chartViews/:id",
-    async (req, res, trx) => {
-        const id = expectInt(req.params.id)
-
-        type ChartViewRow = Pick<
-            DbPlainChartView,
-            "id" | "name" | "updatedAt"
-        > & {
-            lastEditedByUser: string
-            chartConfigId: string
-            configFull: JsonString
-            configPatch: JsonString
-            parentChartId: number
-            parentConfigFull: JsonString
-            queryParamsForParentChart: JsonString
-        }
-
-        const row = await db.knexRawFirst<ChartViewRow>(
-            trx,
-            `-- sql
+}
+
+export async function getChartViewById(
+    req: Request,
+    res: e.Response<any, Record<string, any>>,
```

Comment:
Not specific to this route, just noticed it here. Why are we using `Request` from our `authentication` module, which is just — IMO — an unnecessary alias for `express.Request` and `Response` from Express, when we have our own `Response` in the same `authentication` module with additional properties? If anything it should be the other way around, right?

My preference would be to use `Request` and `Response` only from Express, if possible.